apis/v1alpha1/instrumentation_webhook.go

```diff
@@ -207,7 +238,19 @@ func (r *Instrumentation) validate() error {
 				return fmt.Errorf("spec.sampler.argument should be in rage [0..1]: %s", r.Spec.Sampler.Argument)
 			}
 		}
-	case AlwaysOn, AlwaysOff, JaegerRemote, ParentBasedAlwaysOn, ParentBasedAlwaysOff, XRaySampler:
+	case JaegerRemote, ParentBasedJaegerRemote:
+		// value is a comma separated list of endpoint, pollingIntervalMs, initialSamplingRate
+		// Example: `endpoint=http://localhost:14250,pollingIntervalMs=5000,initialSamplingRate=0.25`
+		if r.Spec.Sampler.Argument != "" {
+			err := parseJaegerRemoteSamplerArgument(r.Spec.Sampler.Argument)
+
+			if err != nil {
+				return fmt.Errorf("spec.sampler.argument is not a valid JaegerRemoteSampler argument: %s", err.Error())
```

Comment:
Is only ParentBasedJaegerRemote allowed to have an arg?  Does JaegerRemote need included in this case?

Reply:
Both are allowed to have argument value. Both are covered in the case statement above. Can you help me understand where else it needs to be included?

Reply:
I ask because the case seems to change the behavior of `JaegerRemote`.  Before is was in a nothing case, but now if a user uses JaegerRemote and supplies arguments it will drop into the `parseJaegerRemoteSamplerArgument` function.  Is that what should happen?  Is the current behavior of doing no validation for JaegerRemote a bug?

Reply:
> Is that what should happen? Is the current behavior of doing no validation for JaegerRemote a bug?

Yes, I don't know if we should call it a bug. When `spec.sampler.argument` is provided for `(parentbased_)jaeger_remote`, it's good to have a  basic validation as we do for the `TraceIDRatio`.